### DISCUSSION TITLE
Settings for Octopus agile import and fixed export

### DISCUSSION THREAD

**Discussion by stefan-west1** [springfall2008/batpred#232] [General]
Hi all. I'm completely new to this, so struggling a little with all the different settings. I have both solar and battery, and I'm currently with octopus agile for my import, and octopus fixed for the export. What I want to set up is basically for the system to charge from the mains when the price is the cheapest overnight, and not worry too much about export. My export rate is fixed at 15p per kwh, so if I can charge at less than that, to me it makes sense to fill the battery from the mains, and just allow all solar to export. However, I'm struggling to work out which things should be on, which should be off etc.

**Comment by gcoan** [comment]
Hi Stefan, welcome to predbat!
I see this was a couple of weeks ago, do you still need help?
Basically follow the install instructions in the documentation repository https://springfall2008.github.io/batpred/installation-summary/

and watch the videos for the setup sequence.  

For specific tariff types, this document https://springfall2008.github.io/batpred/configuration-guide/  describes the settings to use depending on what type of tariff you have. 

Whilst you are getting to grips with what predbat is doing, suggest you set read-only mode on so it doesn't make any changes to your system.

And use the HTML predbat plan to see what its planning on doing
